client/state/rewind/retention/actions.ts

```diff
@@ -0,0 +1,18 @@
+import { Action } from 'redux';
+import { JETPACK_BACKUP_RETENTION_UPDATE } from 'calypso/state/action-types';
+import type { RetentionPeriod } from './types';
+import 'calypso/state/data-layer/wpcom/sites/rewind/retention';
+
+type RequestActionType = Action< typeof JETPACK_BACKUP_RETENTION_UPDATE > & {
+	siteId: number | null;
+	retentionDays: RetentionPeriod;
+};
```

Comment:
This type seems repeated between here and the data layer. Should we declare at a single location and reuse?

Reply:
Moved this type to `retention/types.ts`